src/ResourceManager/Insights/Commands.Insights/OnetimeGenerated/Insights/MetricDefinitionOperations.cs

```diff
@@ -0,0 +1,505 @@
+// 
```

Comment:
There were several folders for metrics in the previous SDK intended to modify and/or extend the metrics functionality beyond what we were able to specify in Hydra.
What I did here was just to copy all files and folders related to mertics into this project so that I can use them to keep metrics working as before. I kept the structure of the files and folders untouched since this is a temporary change.